pkg/cmd/server/kubernetes/master.go

```diff
@@ -380,12 +315,14 @@ func (c *MasterConfig) RunGarbageCollectorController(client *osclient.Client, co
 
 	config = restclient.AddUserAgent(config, "generic-garbage-collector")
 	config.ContentConfig.NegotiatedSerializer = serializer.DirectCodecFactory{CodecFactory: metaonly.NewMetadataCodecFactory()}
+	// TODO: should use a dynamic RESTMapper built from the discovery results.
+	restMapper := registered.RESTMapper()
```

Comment:
answering my own question... we were using dynamic.LegacyAPIPathResolverFunc before, so I don't think so